### PR TITLE
(Work in progress) Adding build docs

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: build_dirs/build_docs.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+import tarfile
+import json
+
+BASE = os.path.abspath(os.path.dirname(__file__))
```
Comment (diraol) [inline build_dirs/build_docs.py]: @macartur let's try to prioritize the use of PathLike objects for dealing with paths/dirs/files.

Can you please rewrite this PR considering this issue? Thanks! =)

**Review by diraol — CHANGES_REQUESTED** [review]

File: build_dirs/build_docs.py (hunk 1 above)
Comment (macartur) [reply build_dirs/build_docs.py]: Done.

File: build_dirs/build_docs.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+from pathlib import Path
+
+
+BASE = Path(os.path.abspath(os.path.dirname(__file__)))
```
Comment (diraol) [inline build_dirs/build_docs.py]: Just check before changing, but I think that
`Path(os.path.abspath(os.path.dirname(__file__)))` == `Path(__file__).parent`

**Review by diraol — DISMISSED** [review]

**Comment by beraldoleal** [comment]
@macartur @diraol Hi guys... there is no automatic way to do that ? Like a integration already builtin on sphinx ?

**Comment by diraol** [comment]
@macartur have you looked at this sphinx extensions?
https://robpol86.github.io/sphinxcontrib-versioning

**Comment by beraldoleal** [comment]
@diraol good.. maybe there are others..

**Comment by macartur** [comment]
So, I saw this sphinx extension, but the script `build_docs.py` is creating a folder with the structure below for all projects with the folder docs.

The structure of docs folder will be:
- docs
   - bethania
       - kytos
       - python-openflow
   - chico
        - kytos
        - python-openflow

And this is made locally using the tarballs stored in each release tag.
I will read more to find others sphinx extensions and a better way to do this.

**Comment by macartur** [comment]
I found a nice package to build the docs: https://buildthedocs.pietroalbini.org/1.0/quickstart/, this seems better than https://robpol86.github.io/sphinxcontrib-versioning.

**Comment by beraldoleal** [comment]
@macartur @diraol so, what is the status of this ?

**Comment by macartur** [comment]
I am trying to do this issue using the buildthedocs package, but I had a problem with templates and styles.

**Comment by macartur** [comment]
I will close this PR, each project will manage your sphinx version using sphinxcontrib-versioning.

**Pull request closed without merging**